NitroxClient/GameLogic/Bases/BuildThrottlingQueue.cs

```diff
@@ -28,7 +28,7 @@ public bool NextEventRequiresFreshFrame()
 
         public void EnqueueBasePiecePlaced(BasePiece basePiece)
         {
-            Log.Info("Enqueuing base piece to be placed techtype: " + basePiece.TechType + " id: " + basePiece.Id + " parentId: " + basePiece.ParentId.OrElse(null) + " build index: " + basePiece.BuildIndex );
+            Log.Info("Enqueuing base piece to be placed techtype: " + basePiece.TechType + " id: " + basePiece.Id + " parentId: " + basePiece.ParentId.OrElse(null) + " build index: " + basePiece.BuildIndex);
```

Comment:
Switch to consistent style, might want to check this is write as written on mobile 
```suggestion
            Log.Info($"Enqueuing base piece to be placed techtype: {basePiece.TechType} id: {basePiece.Id} parentId: {basePiece.ParentId.OrElse(null)} build index: {basePiece.BuildIndex}");
```